eng/PoliCheckExclusions.xml

```diff
@@ -0,0 +1,3 @@
+<PoliCheckExclusions>
```

Comment:
Is this because 3.1.0 hasn't been merged in to master as yet?

Reply:
We don't want to have 3.1 -> master as well as 3.1 -> blazor-wasm -> master. Therefore, this is how 3.1 gets merged into master.